ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphPage.tsx

```diff
@@ -166,9 +166,9 @@ function NetworkGraphPage() {
         clusterFromUrl,
         namespacesFromUrl,
         deploymentsFromUrl,
-        deploymentCount,
         remainingQuery,
         timeWindow,
+        deploymentCount,
```

Comment:
i thought i could remove this but turns out i need it to ensure the graph gets loaded on refresh 🤷🏼‍♀️